public/app/features/dashboard-scene/pages/DashboardScenePage.tsx

```diff
@@ -68,9 +69,15 @@ export function DashboardScenePage({ match, route, queryParams, history }: Props
 
   if (!dashboard) {
     return (
-      <Page layout={PageLayoutType.Canvas} data-testid={'dashboard-scene-page'}>
-        {isLoading && <PageLoader />}
-        {loadError && <h2>{loadError}</h2>}
+      <Page navId="dashboards/browse" layout={PageLayoutType.Canvas} data-testid={'dashboard-scene-page'}>
```

Comment:
🤔  what is the purpose of adding this navId `<Page navId="dashboards/browse" `

Reply:
@axelavargas without it we get no app chrome (nav bar) on full page reload, and we get no breadcrumbs 

the navId is important for breadcrumbs and for AppChrome to know if the route is chromeless or not

Reply:
I see, so this was a missing property we had, thanks Torkel ❤️